test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -1126,33 +1144,35 @@ instance ( Eq (Class.TableConfig h)
       Unions{}         -> OEither $ bimap OId (const OTable) result
       RemainingUnionDebt{} -> OEither $ bimap OId OId result
       SupplyUnionCredits{} -> OEither $ bimap OId OId result
+      SupplyPortionOfDebt{} -> OEither $ bimap OId OId result
```

Comment:
Ah, here we're doing some kind of semi-aligned style. I'm not fan of aligning for this reason, but if we do it in the rest of the module, maybe we should also do it here. Same in in the `IOSim` instance.